Dockerfile

```diff
@@ -1,6 +1,9 @@
 FROM alpine:3.3
 
-RUN apk --no-cache add bash curl jq wget
+RUN (apk add --update bash curl jq wget groff less python py-pip &&\
```

Comment:
When using alpine 3.3 or greater, you can now use `apk --no-cache` instead of `apk add --update` and `rm -rf /var/cache/apk/*` as the new `--no-cache` parameter will only temporarily download the apk cache data and it is automatically purged once complete.